examples/src/iframe/index.mjs

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
If the import isn't used, just delete the import? 😄

Reply:
I assumed there were defined in the global context to be used by an injected script so didn't want to remove it just in case 😅